### DISCUSSION TITLE
3D LUT format

### DISCUSSION THREAD

**Discussion by Benik3** [awawa-dev/HyperHDR#272] [Q&A] [answered]
Hello.
I searched the github, but I didn't find, what format is the 3D LUT file (.3d).
Is possible to use also standard 3D LUT files generated e.g. by DisplayCAL? Or convert them?

Thank you

**Comment by awawa-dev** [comment, marked as answer]
Hi
You can import 3dl Autodesk lut table and convert it to .3d using LUT generator page in HyperHDR app.
Just click 'import' and then 'save' on the form.

**Reply by Benik3** [reply]
Thank you!
